### PR TITLE
New enhancements to the Accounts page

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812.

File: docs/manage/manage-subscription/cloud-flex-credits-accounts.md [hunk 1]
```diff
@@ -115,7 +115,8 @@ The usage forecast gives you an opportunity to see the potential end of the curr
 ### Usage Details by Category
 
 * **Usage Details.** Shows graphs that represent detail usage of Sumo Credits by ingest, storage, or any type of data service utilized from the start of the contract period.
-* **Usage Categories**. Table displaying the credits used, % of credits used, and units by usage categories, ingest type and promotional credits usage.
+* **Usage Categories**. Table displaying the credits used, % of credits used, usgae % change, and units by usage categories ingest type and promotional credits usage.
```
Comment (kimsauce) [inline docs/manage/manage-subscription/cloud-flex-credits-accounts.md:118]:
```suggestion
* **Usage Categories**. Table displaying the credits used, % of credits used, usage % change, units by usage categories ingest type, and promotional credits usage.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
one fix

File: blog-service/2024-03-01-manage.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: New Enhancements for Accounts page (Manage)
```
Comment (kimsauce) [inline blog-service/2024-03-01-manage.md:2]:
```suggestion
title: Accounts page enhancements (Manage)
```

File: blog-service/2024-03-01-manage.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - accounts
+  - release notes
```
Comment (kimsauce) [inline blog-service/2024-03-01-manage.md:6]: No need to put 'release notes'
```suggestion
  - accounts
  - manage account
  - cloud flex credits
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**